<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Practica 1       --- Simulacion de un aeropuerto ---</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnIniciar">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Iniciar simulacion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSiguiente">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(6, 63, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Siguiente Turno</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Datos para iniciar simulacion</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>31</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Symbola</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Numero de aviones:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAviones">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 101, 164);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>31</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Symbola</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Numero de turnos:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTurnos">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>31</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 101, 164);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>70</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Symbola</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Numero de escritorios:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>70</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Symbola</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;NÃºmero de estaciones:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEstaciones">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>69</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 101, 164);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEscritorios">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 101, 164);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblFondo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1020</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagen.qrc">:/aeropuerto.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CustomView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>601</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="consola">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Consola de simulacion de turnos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Imagen de simulacion actual&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>lblFondo</zorder>
   <zorder>btnIniciar</zorder>
   <zorder>btnSiguiente</zorder>
   <zorder>groupBox</zorder>
   <zorder>graphicsView</zorder>
   <zorder>consola</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomView</class>
   <extends>QGraphicsView</extends>
   <header>customview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagen.qrc"/>
 </resources>
 <connections/>
</ui>
